net/golioth/golioth.c

```diff
@@ -498,6 +498,32 @@ int golioth_lightdb_set(struct golioth_client *client, const uint8_t *path,
 	return golioth_send_coap_payload(client, &packet, data, data_len);
 }
 
+int golioth_lightdb_delete(struct golioth_client *client, const uint8_t *path)
+{
+	struct coap_packet packet;
+	uint8_t buffer[GOLIOTH_COAP_MAX_NON_PAYLOAD_LEN];
+	int err;
+	uint8_t *data = "";
+	uint16_t data_len = sizeof(data);
+
+	err = coap_packet_init(&packet, buffer, sizeof(buffer),
+			       COAP_VERSION_1, COAP_TYPE_CON,
+			       COAP_TOKEN_MAX_LEN, coap_next_token(),
+			       COAP_METHOD_DELETE, coap_next_id());
+	if (err) {
+		return err;
+	}
+
+	err = coap_packet_append_option(&packet, COAP_OPTION_URI_PATH,
+					path, strlen(path));
+	if (err) {
+		LOG_ERR("Unable add uri path to packet");
+		return err;
+	}
+
+	return golioth_send_coap_payload(client, &packet, data, data_len);
```

Comment:
```suggestion
	return golioth_send_coap(client, &packet);
```

No reason to send any payload.

Reply:
I didn't see that function. Thanks!